src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -10249,6 +10249,18 @@ private static void payActionCost(final String responseText) {
           skillSuccess = true;
         }
         break;
+
+      case SkillPool.PERPETRATE_MILD_EVIL:
+        if (
+        // Spookypocket
+        responseText.contains("so discombobulated")
+            ||
+            // no items
+            responseText.contains("even mildly evil")
```

Comment:
My understanding is that if there are no items missing it doesn't spend a charge

Reply:
I've tested and it definitely does spend a charge. The message does indicate that perhaps it shouldn't.